opm/simulators/flow/FlowMainEbos.hpp

```diff
@@ -271,33 +270,9 @@ namespace Opm
             }
         }
 
-    protected:
-        void setupParallelism()
```

Comment:
This function seems unchanged, why is it moved?

Reply:
It is the printPRTHeader that is moved to public.

Reply:
I see, thanks. Merging!